src/components/LanguageButton/LanguageButton.js

```diff
@@ -1,13 +1,18 @@
 import React from "react";
```

Comment:
if this component is used only inside language select screen probably it's better to hold it inside language select folder

Reply:
It’s also used in settings screen

Reply:
deleted it from the components anyway